po/reuse-docs.pot

```diff
@@ -0,0 +1,1858 @@
+# SOME DESCRIPTIVE TITLE
+# Copyright (C) YEAR Free Software Foundation, Inc.
```

Comment:
From what I understand, this file is created automatically based on the MD file. Any chance that we get a new comment line with `SPDX-License-Identifier: ...` in it?

Reply:
I think you can add comments to that block.  That file format is GNU _gettext_ if you want a reference.